### PR TITLE
Features/text styled components

### LINKED ISSUE #221: FIX/UI: Switch text elements using Tailwind to styled Components
## The story:
As a **developer** that **wants to use styled components**, I want to **implement styled components in our project**

---

## What we should do:
We should **replace all tailwind styling in text** so that **we are using styled components in the project**.

---

## To be successful we need to:
_Check these off as they are completed:_
- [ ] Check all `<p><h1><h2><h3> and <h4> ` & if these elements are using tailwind styling replace them for styled components.

---

### PR DESCRIPTION
**Pull request by YagoMackay**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

## Fixed issues 
- Fixes #221

## Changes
-  All `<p> <h1> <h2> <h3> and <h4> ` elements are styled with base style or Styled Components.

## PR Checklist
- [x] Does your PR contain relevant tests that pass?
- [x] Is your code linted? `yarn lint` or `yarn lint:fix`
- [x] Does your code build correctly in ALL projects? `yarn build`

### CODE REVIEW

File: packages/client/src/components/ProgressWidget/ProgressPath.tsx [hunk 1]
```diff
@@ -13,6 +14,16 @@ interface PathProgress {
   progressStroke?: string;
 }
 
+const Styledp = styled.p`
+font-size: .875rem;
+text-align: center;
+margin-bottom: 2rem;
+font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+font-weight: 800;
+letter-spacing: 0.1em;
```
Comment (tristanMatthias) [inline packages/client/src/components/ProgressWidget/ProgressPath.tsx:23]: Can we use theme values for all this? It feels quite verbose haha

File: packages/client/src/pages/Login/Login.page.tsx [hunk 2]
```diff
@@ -6,8 +6,20 @@ import { getGQLError } from '@codement/ui/lib/apollo';
 import Logo from '@codement/ui/images/logo.svg';
 import People from '@codement/ui/images/welcome-people.svg';
 import { LocalStorage } from '@codement/ui/lib/localStorage';
+import styled from 'styled-components';
 import { Page } from '../../components/Page/Page';
 
+const Styledp = styled.p`
+margin-top: 0.5rem;
+margin-bottom: 1rem;
+font-size: .875rem;
```
Comment (tristanMatthias) [inline packages/client/src/pages/Login/Login.page.tsx:15]: Theme values here too please 🙏

File: packages/client/src/pages/Onboarding/OnboardingStep.tsx [hunk 3]
```diff
@@ -20,6 +21,17 @@ export interface OnboardingStepProps {
...
 
+const Styledp = styled.p`
+font-size: 1.125rem;
+text-align: center;
+margin-bottom: 2rem;
+`;
+
+const Title = styled.h3`
+white-space: pre-wrap;
+margin-bottom: 0.75rem;
```
Comment (tristanMatthias) [inline packages/client/src/pages/Onboarding/OnboardingStep.tsx:32]: And here for the margins

File: packages/ui/css/base.ts [hunk 4]
```diff
@@ -46,6 +46,11 @@ html {
     letter-spacing: 0.05em;
     color: ${props => props.theme.colors.grey[700]};
   }
+
+  h4.titleCentred {
+  margin-bottom: 1rem;
+  text-align: center;
+  }
```
Comment (tristanMatthias) [inline packages/ui/css/base.ts:53]: We should probably not do any global classes like this as we're moving away from that style of dev. Mind if we turn this into a component instead?

**Review by tristanMatthias — CHANGES_REQUESTED** [review]
Hey @YagoMackay, thanks for the thoroughness! As for many of the styled component PRs coming through, just some changes around using the theme variables instead of hardcoding.
Cheers dude!

**Comment by YagoMackay** [comment]
Thanks for everything @tristanMatthias ! I'll make sure to switch them to theme variables.

**Comment by tristanMatthias** [comment]
Hey @YagoMackay! I used this PR in #239 as a base. Thanks for this! Made it much easier :)

**Pull request closed without merging**